src/check/parse/IR.zig

```diff
@@ -1778,7 +1801,24 @@ pub const NodeStore = struct {
         statements: []const StatementIdx,
         /// The token that represents the newline preceding this block, if any
         whitespace: ?TokenIdx,
+
         region: Region,
+
+        pub fn toSExpr(self: @This(), gpa: Allocator, env: *base.ModuleEnv, ir: *IR) anyerror!sexpr.Node {
+            var children = std.ArrayList(sexpr.Node).init(gpa);
+
+            for (self.statements) |stmt_idx| {
+                const stmt = ir.store.getStatement(stmt_idx);
+                try children.append(try stmt.toSExpr(gpa, env, ir));
+            }
+
+            return .{
+                .node = .{
+                    .value = "body",
```

Comment:
Lastly, we have to decide if the node owns the strings that are passed into it. Likely the answer should be yes.

So here and also for all of the `.string` nodes, we have to do:
```suggestion
                    .value = try gpa.dupe(u8, "body"),
```